app/assets/stylesheets/application.scss

```diff
@@ -57,7 +57,6 @@ li { margin: 0; }
 
  img {
    display: inline;
-   margin: 5px 10px;
```

Comment:
This looks to be for all image tags across the site. Are you sure that removing these margins wont affect any existing images? It might be safer to set a margin 0 on an exisiting image class for the specific images you want to have that zero value.

Reply:
Yeah sorry about that one. Added back in now.